specification/cognitiveservices/data-plane/Speech/BatchAvatar/stable/2024-08-01/batchavatar.json

```diff
@@ -0,0 +1,848 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Batch synthesis API",
+    "description": "Batch synthesis API for text to speech avatar.",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "url": "https://docs.microsoft.com/azure/cognitive-services/speech-service/support"
+    },
+    "version": "2024-08-01"
+  },
+  "paths": {
+    "/batchsyntheses": {
+      "get": {
+        "tags": [
+          "BatchSynthesis"
+        ],
+        "description": "Gets the list of batch syntheses.",
+        "operationId": "BatchSyntheses_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/QuerySkip"
+          },
```

Comment:
Just a caution, that you are not going to remove this `skip` query param in future.

Reply:
yes, we will have `skip` in all future versions